clusterloader2/docs/oom_detection.md

```diff
@@ -0,0 +1,47 @@
+# Clusterloader2 - tracking OOMs
+
+In order to track pods OOMs (out of memory) during clusterloader2 tests
+execution, TestMetrics has been augmented with `ClusterOomsTracker` measurement.
+This outputs an appropriate summary containing basic information regarding OOMs
+inside the cluster like the PID of OOMing process or the name of a node where
+the OOM happened.
```

Comment:
I think I wrote it somewhere already, but please mention it explicitly here that:
given that oom-detector is based on k8s events and events are best-effort by their nature, the report is not guaranteed to accurately describe what really happened (some ooms may be missed)